libgcc/config.host

```diff
@@ -454,13 +454,16 @@ aarch64*-*-vxworks7*)
 	tmake_file="${tmake_file} ${cpu_type}/t-softfp t-softfp t-crtfm"
 	tmake_file="${tmake_file} t-dfprules"
 	;;
-aarch64-*-mingw*)
+aarch64*-*-mingw*)
 	case ${target_thread_file} in
 	  win32)
-	    tmake_file="$tmake_file i386/t-gthr-win32"
+	    tmake_thr_file="i386/t-gthr-win32"
 	    ;;
 	  posix)
-	    tmake_file="i386/t-mingw-pthread $tmake_file"
+	    tmake_thr_file="i386/t-mingw-pthread"
```

Comment:
please merge with the latest changes. are these changes really needed for posix and mcf now? is it tested somehow?

Reply:
Yes, as stated in the description, this had to be rebased on #5. Since #7 implemented more or less the same changes and it's merged now, this PR can be rebased (and already is) to the current `woarm64`branch.